bank2ynab.conf

```diff
@@ -12,6 +12,7 @@ Use Regex for Filename = False
 Source CSV Delimiter = ,
 Header Rows = 1
 Footer Rows = 0
+Discard Invalid Dates = False
```

Comment:
This is interesting. I can't think of any reason where we *wouldn't* want to discard invalid dates! I wonder is this flag necessary.